src/internal/clusterstate/v2.8.0.go

```diff
@@ -0,0 +1,12 @@
+package clusterstate
+
+import (
+	"github.com/pachyderm/pachyderm/v2/src/internal/migrations"
+
+	v2_8_0 "github.com/pachyderm/pachyderm/v2/src/internal/clusterstate/v2.8.0"
+)
+
+var state_2_8_0 migrations.State = v2_8_0.Migrate(state_2_7_0)
+
+// DO NOT MODIFY THIS STATE
```

Comment:
This comment isn't really accurate.  It doesn't mean anything if we include it in the place where we can add migrations.